src/main/resources/default/assets/common/scripts/overlay.js.pasta

```diff
@@ -106,7 +104,19 @@
             sirius.executeEmbeddedScripts(_overlay);
         }
 
-        sirius.dispatchEvent('sci-overlay-shown');
+        sirius.dispatchEvent('sci-overlay-created');
+    }
+
+    /**@
+     * Creates and shows a new overlay.
+     *
+     * If one or more overlays already exist, the new overlay will be shown on top of already existing overlays.
+     */
```

Comment:
Copy & Paste error: JS-doc doesn't match function. A new overlay is neither created nor can there be more overlays.